Source/WebCore/platform/graphics/ImageOrientation.h

```diff
@@ -75,7 +75,7 @@ struct ImageOrientation {
     }
 
     constexpr operator int() const { return static_cast<int>(m_orientation); }
-    friend constexpr bool operator==(const ImageOrientation& lhs, const ImageOrientation& rhs) { return lhs.m_orientation == rhs.m_orientation; }
+    friend constexpr bool operator==(const ImageOrientation&, const ImageOrientation&) = default;
     friend constexpr bool operator==(const ImageOrientation& lhs, const Orientation& rhs) { return lhs.m_orientation == rhs; }
```

Comment:
Is this overload needed? Does removing it cause a compilation failure?

Reply:
Doesn't seem needed, will remove.